Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,9 +115,30 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
+
+    // Parse command line options
+    String requestedURL = ""_s;
```

Comment:
This is `WTF::String`. WTF isn't public API. Don't use WTF in MiniBrowser. use std::string instead.